docs/topics/forms/index.txt

```diff
@@ -206,10 +206,10 @@ will contain the form data.
 Now you'll also need a view corresponding to that ``/your-name/`` URL which will
 find the appropriate key/value pairs in the request, and then process them.
 
-This is a very simple form. In practice, a form might contain dozens or
-hundreds of fields, many of which might need to be pre-populated, and we might
-expect the user to work through the edit-submit cycle several times before
-concluding the operation.
+In practice, a form might contain dozens or hundreds of fields instead of just
```

Comment:
I think this can be reverted, the missing sentence makes the rest read weirdly because it's only the second half of an implicit comparison.